### PR TITLE
Store Data Locally

### PR DESCRIPTION
**Pull request by Abrar052**
### Store Data Locally

- Added function in index.js to save data locally
- Added Google Tag Manager Script to save forms Data
- Added Google Tag Manager ID in the Script
- Added a function of Google Tag Manager to Save Data

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=GTM-WNXMSPC"></script>
+
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){window.dataLayer.push(arguments);}
+        gtag('js', new Date());
+      
+        gtag('config', 'GTM-WNXMSPC');
+      </script>
```
Comment (Ugboaja-Uchechi) [inline index.html:20]: - [ ] Please make sure you store data gotten from the contact form is stored in local storage as mentioned in the project requirement  🙏 .  You can learn more about how to use local storage from this [video](https://youtu.be/AUOzvFzdIk4)

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Abrar052 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abrar052) [reply index.html:20]: @Ugboaja-Uchechi thank you for the resource.

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: 

Hi @Abrar052, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Descriptive PR :heavy_check_mark: 
- [ ] Good implementation of change required :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**